test/test_helper.rb

```diff
@@ -46,6 +46,11 @@ class ActionDispatch::IntegrationTest
 
   def setup
     stub_shared_component_locales
+    # FIXME
+    # We need to disable errors for JavaScript since Static is mocked in our tests.
+    # and jQuery is pulled in from Static, this means we're not currently testing
+    # webchat, this should be resolved even when we remove this after the AB test.
+    page.driver.browser.js_errors = false
```

Comment:
Will this break any accessibility errors that the component guide raises?
The component guide raises JS errors when it runs the aXe suite if it finds accessibility problems.

Reply:
(We are building components at the moment and will rely on these tests working)

Reply:
@fofr I'm under the impression that this setting only suppresses errors from the Minitest tests. Will double check with @nickcolley

Reply:
I think so yes, maybe we could only include the javascript on the interstitial page, and only turn the errors off on that page.

I'm not happy with most of the approach here, unfortunately we have a fixed deadline since we can only test within an agreed window of time based on conversations with HMRC etc